extensions/3rdparty/LyricWiki/LyricFind/LyricFindTrackingService.class.php

```diff
@@ -28,7 +28,7 @@ class LyricFindTrackingService extends WikiaService {
 	private function markLyricForRemoval($pageId) {
 		$this->wf->SetWikiaPageProp(WPP_LYRICFIND_MARKED_FOR_REMOVAL, $pageId, 1);
 
-		self::log(__METHOD__, "marked page #{$pageId} for removal");
+		$this->info( __METHOD__, [ 'page_id' => $pageId ] );
```

Comment:
I'd go with something more like `__CLASS__ - marked page ....` so the text messages would be meaningful